test/ProductConstructionService.DependencyFlow.Tests/SubscriptionUpdaterTests.cs

```diff
@@ -64,7 +64,7 @@ private void ThenUpdateAssetsAsyncShouldHaveBeenCalled(UpdaterId forUpdater, Bui
                     SourceRepo,
                     NewCommit,
                     Capture.In(updatedAssets),
-                    true));
+                    true)==null);
```

Comment:
This looks suspicious. It possibly hints the parameters where not matched and a `null` Task was returned. This means we didn't match the parameters correctly and might hint at a break in the test (e.g. arguments changed).